src/test/java/org/jboss/xavier/analytics/rules/workload/inventory/complexity/ComplexityTest.java

```diff
@@ -263,7 +263,7 @@ public void testMoreThanOneFlagUndetectedOS() {
         facts.put("agendaGroup", "Complexity");
 
         WorkloadInventoryReportModel workloadInventoryReportModel = new WorkloadInventoryReportModel();
-        workloadInventoryReportModel.setOsDescription(null);
+        workloadInventoryReportModel.setOsDescription("Not detected");
```

Comment:
Shouldn't we use here the value of the constant WorkloadInventoryReportModel.OS_NAME_DEFAULT_VALUE ?

Reply:
Done